nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -1863,10 +1863,9 @@ void nxagentKeycodeConversionSetup(void)
     fprintf(stderr, "Failed to create the keyboard file\n");
   }
 
-  if (nxagentOption(ClientOs) == ClientOsLinux &&
-            drules != NULL && dmodel != NULL &&
-                (strcmp(drules, "evdev") == 0 ||
-                    strcmp(dmodel, "evdev") == 0))
+  if (drules != NULL && dmodel != NULL &&
```

Comment:
One more comment: _if_ FreeBSD uses evdev it might be compatible to the Linux version (at least i'd expect this for identically named components). So again no reason to check the remote OS.